adapters/repos/db/inverted/searcher_ref_filter.go

```diff
@@ -245,3 +241,13 @@ func (r *refFilterExtractor) validate() error {
 
 	return nil
 }
+
+func determineRefLimit() int {
+	limit, err := strconv.ParseInt(os.Getenv("QUERY_REF_LIMIT"), 10, 64)
+	if err != nil {
+		limit = config.DefaultQueryReferenceLimit
+	} else if limit < 0 {
```

Comment:
what if limit == 0? is it unlimited or 0?

Reply:
If limit == 0, no refs will be returned. This was requested by @kcm but I am open to suggestions

Reply:
thx, it is clear now. I would just add a comment that 0 is allowed and works exactly as 0 (nothing is returned)

Reply:
ok, adding the comment 👍